vagrant-post-script/orchestrate_docker.sh

```diff
@@ -11,7 +11,7 @@ chmod +x helper.sh
 find . -type f -name "*.sh" -exec dos2unix {} \+;
 dos2unix .env Makefile
 
-if [[ -f .env ]]; then
+if [[ -f .init ]]; then
```

Comment:
the file .init is never created anywhere

Reply:
https://github.com/AEGEE/MyAEGEE/blob/master/helper.sh#L74 as part of make init